src/FakeItEasy/Core/FakeAsserter.cs

```diff
@@ -61,7 +61,21 @@ private static void AppendExpectation(IEnumerable<IFakeObjectCall> calls, string
 
             if (calls.Any())
             {
-                writer.Write("but found it #{0} times among the calls:", matchedCallCount);
+                switch (matchedCallCount)
+                {
+                    case 0:
+                        writer.Write("but was not found among the calls:");

```

Comment:
Switching to the passive voice for this case is a bit strange... Maybe _"but didn't find it among the calls:"_ instead?

Reply:
@thomaslevesque I chose "was not" over "didn't" since it's used more often in the whole solution
But I can also use "didn't" :-)
![image](https://user-images.githubusercontent.com/9402861/33828821-e291a40c-de6d-11e7-900d-b67fbd819d0a.png)
![image](https://user-images.githubusercontent.com/9402861/33828827-e6318622-de6d-11e7-8fe3-289ea9f0b360.png)

BTW how do I update a PR from my changes in VS?

Reply:
I just noticed that a lot of those are "was not matched" and maybe those should be changed to "didn't match", too

Reply:
> But I can also use "didn't" :-)

Let's leave @blairconrad a chance to review as well. He's a native English speaker, which I'm not ;)

> BTW how do I update a PR from my changes in VS?

You just push to the same branch. You can either add a commit, or amend the first one and force push.

Reply:
@thomaslevesque Since he thumbed up, I have changed it.
Thanks for the hint, didn't expect things to be this comfortable in git :-)